examples/ML-DSA-65.crt.txt

```diff
@@ -0,0 +1,251 @@
+fbfee604abb7379091ea8e6c5c74dfc0283666b40c0793870028204a136bf5da
+9568eb798d349038bdb0c11e03445e7847cb5069c75cf28ac601c7799d958210
+ddbcb226e51afef9f1de47b073873d6d3f97456bede085082e74a298b2cd48f4
+b3093155f366c8fa601c6af858dfa32c08491b2a29887f90335949a5d6edaa67
+9882a3a95d6bf6d970a221f4b9d3d8cbf384af81aac95e2b3294e04789ac8372
+7a5dc04559f96af41d8a053516feeeebc52746eb6ab2819e09108710d835f011
+fa63065872ad334d5cdffb2b2310507e92fc993ae317da97f4f309cdaf0f67ed
+99d90215576083849f953b246d7fedb3fdb67679850a5ad404e64147fb7cf4f6
+aeddd05afb4b834968d1fe88014960dce5d942236526e12a478d69e5fbe69703
+10b308c06845018cfc7b2ab430a13a6b1ac7bb02cccbb3d911ac2f11068613fb
+e029bfdce02cf5cd38950ed72c83944edfbc75615af87f864c051f3c55456c54
+12863a40c06d1dab562bdff0571b8d3c3917bbd300880bba5e998239b95fa91b
+7d6416d4f398b3adbcd30983ed3592b4d9ef7d4236fd00f50d98aa53a235ac41
+72720f77d96172672980cfe8ff7a5a702783edc2ba31b2259015a112fc7f468a
+9c2f9464039002d30ef678b4cb798bc116216bf7a9a7c18ba03b7b58fd07515d
+3115049d3614be7a07e744300750df1d2c58753389059eafc3d785ccdd31c076
+48bedc03a5c3b8ad46d064d59c13d57374729fc4e295362e2a5191204530428b
+c1522afa28ff5fe1655e304ca5bc8c27ad0e0c6a39dd4df28956c14b38cc9368
+2cefe402bbd5e82d29c464e44eb5d37b48fc568dfe0cc6e8e16baea05e513559
+0f19294e73e8367b0216dbb815030b9de55913f08039c42351c59e5515dd5af8
+e089a15e625e8f6dee639386c46497d7a263288774de581a7de9629b41b44241
+41f978fb8331208efdec3c6e0de39bc57063f3dcd6c470373c08891ea29cbc7c
+c6d6483b8889083ace86aa7b51b1c2cfe6e2ad18d97ce36fbc56ea42fae97e6a
+7ac114864478c366df1ebb1e7b11a9098504fd5975bdf1f49dc70002b63c1739
+a9d263fbad4073f6a9f6c2b8af4b4c332a103a0cffa5deeb2d062ca3c215fd36
+0026be7c5164f4a4424ef74948804d66f46487732c8202c795478647b4ea71d6
+27c086024cca354a41f0877b38f19b3774ad2095c8da53b069e21c76ae2d2007
+e16719ed40080d334f7da52e9f5a5990439caf083a95b833f02ad10a08c1a6d0
+f260c007285bd4a2f47703a5aef465287d253b18ac22514316210ff566814b10
+f87a293d6f199d3c3959990d0c1268b4f50d5f9fcefbbf237bd0c28b80182d66
+59741f14f10bfbb21bba12ab620aa2396f56c0686b4ea9017990224216b2fe8a
+d76c4a9148eef9a86a3635a6aa77bc1dcfb6fba59a77dfda9b7530dc0ca8648c
+8d973738e01bab8f08b4905e84aa4641bd602410cd97520265f2f231f2b35e15
+eb2fa04d2bd94d5a77abaf1e0e161010a990087f5b46ea988b2bc0512fda0fa9
+23dadd6c45c5301d09483673265b5ab2e10f4ba520f6bbad564a5c3d5e27bdb0
+80f7d20e13296a3181954c39c649c943ebe17df5c1f7aae0a8fe126c477585a5
+d4d648a0d008b6af5e8cd31be69a9296d4f3fd25ed86f221e4b93f65f5929967
+533624b9235750c30707550b58536d109a7131c5a5bbe4a5715567c12534aec7
+660761eebb9fae2891c774589b80e566ad557ddef7367196b7227ea9870ef09d
+dfec79d6b9319a6879b5205d76bf7aba5acf33afb59d17fc54e68383d6be5a08
+e9b66da53dcde008bb294b8582bd132cdcc49959fdbc21e52721880c8ad0352c
+79f03a43bbd84c4cdfdc6c529005e1e7cd9a349a7168a35569ba5dea818968d5
+a91466bd6e64e20bf62417198afc4e81c28dd77ed4028232398b52fbde86bc84
+f475b9016710ce2aabc11a06b4dbac901ec16cf365ca3f2d53813948a693a0f9
+3e79c46ca5d5a6dca3d28ca50ad18bd13fca55059dd9b185f79f9c47196a4e81
+b2104bc460a051e02f2e8444f` }
```

Comment:
There seems to be a compatibility issue here with `der2ascii`:
```
Syntax error: line 111: encoding/hex: invalid byte: U+000A
```

Reply:
It's because the long lines are wrapped using `fold`. Check out `examples/pretty`.